README.md

```diff
@@ -95,18 +95,18 @@ c2patool -d sample/C.jpg
 
 The tool displays the detailed report to standard output (stdout).
 
-### Information report
+### Displaying an information report
 
-The `--info` option will print a high level report about the file and related C2PA data. 
-For a cloud manifest it will display the url to the manifest.
-For embedded manifests, the size of the manifest store and number of manifests will be displayed. It will also report if the manifest validated or any errors encountered in validation.
+Use the `--info` option to print a high-level report about the asset file and related C2PA data. 
+For a cloud manifest the tool displays the URL to the manifest.
```

Comment:
This changed, for cloud manifests, the tool will now try to retrieve the cloud manifest, and if the fetch succeeds it  will display the same information as embedded manifests, in addition to the cloud url.

Reply:
I'll update this after this PR lands